Gemfile.lock

```diff
@@ -0,0 +1,141 @@
+GEM
```

Comment:
Gemfile is used to lock down dependencies for a ruby environment.  This is mostly for fastlane since fastlane is a ruby app we want to make sure it continues to use the same dependencies across builds. Recommendation from fastlane:  https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile